### ISSUE TITLE
UnicodeDecodeError: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128)

### ISSUE THREAD

**Issue by yweitkemper** [PokemonGoF/PokemonGo-Bot#4216]
### Expected Behavior
Start the bot already tried, 
pip install --upgrade -r requirements.txt
source bin/activate  

And deleting the whole folder and restarting it. 

### Actual Behavior
./run.sh
2016-08-18 07:30:14,258 [       cli] [INFO] PokemonGO Bot v1.0
2016-08-18 07:30:14,262 [       cli] [INFO] Configuration initialized
2016-08-18 07:30:14,262 [pokemongo_bot.health_record.bot_event] [INFO] Health check is enabled. For more information:
2016-08-18 07:30:14,263 [pokemongo_bot.health_record.bot_event] [INFO] https://github.com/PokemonGoF/PokemonGo-Bot/tree/dev#analytics
2016-08-18 07:30:14,279 [requests.packages.urllib3.connectionpool] [INFO] Starting new HTTP connection (1): www.google-analytics.com
2016-08-18 07:30:14,500 [PokemonGoBot] [INFO] [set_start_location] Setting start location.
Traceback (most recent call last):
  File "pokecli.py", line 570, in <module>
    main()
  File "pokecli.py", line 162, in main
    report_summary(bot)
  File "pokecli.py", line 197, in report_summary
    metrics.capture_stats()
  File "/Users/naturalhealingways/PokemonGo-Bot/pokemongo_bot/metrics.py", line 84, in capture_stats
    response_dict = request.call()
  File "/Users/naturalhealingways/PokemonGo-Bot/pokemongo_bot/api_wrapper.py", line 101, in call
    if not self.can_call():
  File "/Users/naturalhealingways/PokemonGo-Bot/pokemongo_bot/api_wrapper.py", line 57, in can_call
    raise NoPlayerPositionSetException()
pgoapi.exceptions.NoPlayerPositionSetException
2016-08-18 07:30:14,605 [sentry.errors] [ERROR] Sentry responded with an error: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128) (url: https://app.getsentry.com/api/90254/store/)
Traceback (most recent call last):
  File "/Users/naturalhealingways/PokemonGo-Bot/lib/python2.7/site-packages/raven/transport/threaded.py", line 174, in send_sync
    super(ThreadedHTTPTransport, self).send(data, headers)
  File "/Users/naturalhealingways/PokemonGo-Bot/lib/python2.7/site-packages/raven/transport/http.py", line 47, in send
    ca_certs=self.ca_certs,
  File "/Users/naturalhealingways/PokemonGo-Bot/lib/python2.7/site-packages/raven/utils/http.py", line 66, in urlopen
    return opener.open(url, data, timeout)
  File "/Users/naturalhealingways/PokemonGo-Bot/lib/python2.7/site-packages/future/backports/urllib/request.py", line 494, in open
    response = self._open(req, data)
  File "/Users/naturalhealingways/PokemonGo-Bot/lib/python2.7/site-packages/future/backports/urllib/request.py", line 512, in _open
    '_open', req)
  File "/Users/naturalhealingways/PokemonGo-Bot/lib/python2.7/site-packages/future/backports/urllib/request.py", line 466, in _call_chain
    result = func(*args)
  File "/Users/naturalhealingways/PokemonGo-Bot/lib/python2.7/site-packages/raven/utils/http.py", line 46, in https_open
    return self.do_open(ValidHTTPSConnection, req)
  File "/Users/naturalhealingways/PokemonGo-Bot/lib/python2.7/site-packages/future/backports/urllib/request.py", line 1284, in do_open
    h.request(req.get_method(), req.selector, req.data, headers)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/httplib.py", line 1057, in request
    self._send_request(method, url, body, headers)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/httplib.py", line 1097, in _send_request
    self.endheaders(body)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/httplib.py", line 1053, in endheaders
    self._send_output(message_body)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/httplib.py", line 895, in _send_output
    msg += message_body
UnicodeDecodeError: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128)
2016-08-18 07:30:14,607 [sentry.errors.uncaught] [ERROR] [u'NoPlayerPositionSetException', u'  File "pokecli.py", line 570, in <module>', u'  File "pokecli.py", line 162, in main', u'  File "pokecli.py", line 197, in report_summary', u'  File "pokemongo_bot/metrics.py", line 84, in capture_stats', u'  File "pokemongo_bot/api_wrapper.py", line 101, in call', u'  File "pokemongo_bot/api_wrapper.py", line 57, in can_call']
Thu Aug 18 07:30:14 PDT 2016 Pokebot  Stopped.
Press any button or wait 20 seconds to continue.
2016-08-18 07:30:35,777 [       cli] [INFO] PokemonGO Bot v1.0
2016-08-18 07:30:35,781 [       cli] [INFO] Configuration initialized
2016-08-18 07:30:35,782 [pokemongo_bot.health_record.bot_event] [INFO] Health check is enabled. For more information:
2016-08-18 07:30:35,782 [pokemongo_bot.health_record.bot_event] [INFO] https://github.com/PokemonGoF/PokemonGo-Bot/tree/dev#analytics
2016-08-18 07:30:35,800 [requests.packages.urllib3.connectionpool] [INFO] Starting new HTTP connection (1): www.google-analytics.com
2016-08-18 07:30:36,062 [PokemonGoBot] [INFO] [set_start_location] Setting start location.
Traceback (most recent call last):
  File "pokecli.py", line 570, in <module>
    main()
  File "pokecli.py", line 162, in main
    report_summary(bot)
  File "pokecli.py", line 197, in report_summary
    metrics.capture_stats()
  File "/Users/naturalhealingways/PokemonGo-Bot/pokemongo_bot/metrics.py", line 84, in capture_stats
    response_dict = request.call()
  File "/Users/naturalhealingways/PokemonGo-Bot/pokemongo_bot/api_wrapper.py", line 101, in call
    if not self.can_call():
  File "/Users/naturalhealingways/PokemonGo-Bot/pokemongo_bot/api_wrapper.py", line 57, in can_call
    raise NoPlayerPositionSetException()
pgoapi.exceptions.NoPlayerPositionSetException
2016-08-18 07:30:36,191 [sentry.errors] [ERROR] Sentry responded with an error: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128) (url: https://app.getsentry.com/api/90254/store/)
Traceback (most recent call last):
  File "/Users/naturalhealingways/PokemonGo-Bot/lib/python2.7/site-packages/raven/transport/threaded.py", line 174, in send_sync
    super(ThreadedHTTPTransport, self).send(data, headers)
  File "/Users/naturalhealingways/PokemonGo-Bot/lib/python2.7/site-packages/raven/transport/http.py", line 47, in send
    ca_certs=self.ca_certs,
  File "/Users/naturalhealingways/PokemonGo-Bot/lib/python2.7/site-packages/raven/utils/http.py", line 66, in urlopen
    return opener.open(url, data, timeout)
  File "/Users/naturalhealingways/PokemonGo-Bot/lib/python2.7/site-packages/future/backports/urllib/request.py", line 494, in open
    response = self._open(req, data)
  File "/Users/naturalhealingways/PokemonGo-Bot/lib/python2.7/site-packages/future/backports/urllib/request.py", line 512, in _open
    '_open', req)
  File "/Users/naturalhealingways/PokemonGo-Bot/lib/python2.7/site-packages/future/backports/urllib/request.py", line 466, in _call_chain
    result = func(*args)
  File "/Users/naturalhealingways/PokemonGo-Bot/lib/python2.7/site-packages/raven/utils/http.py", line 46, in https_open
    return self.do_open(ValidHTTPSConnection, req)
  File "/Users/naturalhealingways/PokemonGo-Bot/lib/python2.7/site-packages/future/backports/urllib/request.py", line 1284, in do_open
    h.request(req.get_method(), req.selector, req.data, headers)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/httplib.py", line 1057, in request
    self._send_request(method, url, body, headers)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/httplib.py", line 1097, in _send_request
    self.endheaders(body)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/httplib.py", line 1053, in endheaders
    self._send_output(message_body)
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/httplib.py", line 895, in _send_output
    msg += message_body
UnicodeDecodeError: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128)
2016-08-18 07:30:36,193 [sentry.errors.uncaught] [ERROR] [u'NoPlayerPositionSetException', u'  File "pokecli.py", line 570, in <module>', u'  File "pokecli.py", line 162, in main', u'  File "pokecli.py", line 197, in report_summary', u'  File "pokemongo_bot/metrics.py", line 84, in capture_stats', u'  File "pokemongo_bot/api_wrapper.py", line 101, in call', u'  File "pokemongo_bot/api_wrapper.py", line 57, in can_call']
Thu Aug 18 07:30:36 PDT 2016 Pokebot  Stopped.
Press any button or wait 20 seconds to continue.

### Your FULL config.json (remove your username, password, gmapkey and any other private info)
  "encrypt_location": "",
    "websocket_server": false,
    "heartbeat_threshold": 10,
    "tasks": [
      {
        "type": "HandleSoftBan"
      },
      {
        "type": "SleepSchedule",
        "config": {
          "enabled": false,
          "time": "22:54",
          "duration":"7:46",
          "time_random_offset": "00:24",
          "duration_random_offset": "00:43"
        }
      },
      {
        "type": "CollectLevelUpReward"
      },
      {
        "type": "IncubateEggs",
        "config": {
          "longer_eggs_first": true
        }
      },
      {
        "type": "UpdateLiveStats",
        "config": {
          "enabled": false,
          "min_interval": 10,
          "stats": ["username", "uptime", "stardust_earned", "xp_earned", "xp_per_hour", "stops_visited"],
          "terminal_log": true,
          "terminal_title": true
        }
      },
      {
        "type": "TransferPokemon",
        "config": {
          "transfer_wait_min": 1,
          "transfer_wait_max": 4
        }
      },
      {
        "type": "NicknamePokemon",
        "config": {
          "enabled": false,
          "nickname_template": "{iv_pct}_{iv_ads}"
        }
      },
      {
        "type": "EvolvePokemon",
        "config": {
          "evolve_all": "none",
          "first_evolve_by": "cp",
          "evolve_above_cp": 500,
          "evolve_above_iv": 0.8,
          "logic": "or",
          "evolve_speed": 20,
          "use_lucky_egg": false
        }
      },
      {
        "type": "RecycleItems",
        "config": {
          "min_empty_space": 15,
          "max_balls_keep": 150,
          "max_potions_keep": 50,
          "max_berries_keep": 70,
          "max_revives_keep": 70,
          "item_filter": {
            "Pokeball":       { "keep" : 100 },
            "Potion":         { "keep" : 10 },
            "Super Potion":   { "keep" : 20 },
            "Hyper Potion":   { "keep" : 30 },
            "Revive":         { "keep" : 30 },
            "Razz Berry":     { "keep" : 100 }
          },
          "recycle_wait_min": 1,
          "recycle_wait_max": 4
        }
      },
      {
        "type": "CatchPokemon",
        "config": {
          "catch_visible_pokemon": true,
          "catch_lured_pokemon": true,
          "min_ultraball_to_keep": 5,
          "catch_throw_parameters": {
            "excellent_rate": 0.1,
            "great_rate": 0.5,
            "nice_rate": 0.3,
            "normal_rate": 0.1,
            "spin_success_rate" : 0.6
          },
          "catch_simulation": {
            "flee_count": 3,
            "flee_duration": 2,
            "catch_wait_min": 2,
            "catch_wait_max": 6,
            "berry_wait_min": 2,
            "berry_wait_max": 3,
            "changeball_wait_min": 2,
            "changeball_wait_max": 3
          }
        }
      },
      {
        "type": "SpinFort",
        "config": {
          "spin_wait_min": 2,
          "spin_wait_max": 3
        }
      },
      {
        "type": "MoveToFort",
        "config": {
          "lure_attraction": true,
          "lure_max_distance": 2000
        }
      },
      {
        "type": "FollowSpiral",
        "config": {
          "diameter": 4,
          "step_size": 70
        }
      }
    ],
    "map_object_cache_time": 5,
    "forts": {
      "avoid_circles": true,
      "max_circle_size": 50,
      "cache_recent_forts": true
    },
    "walk_max": 4.16,
    "walk_min": 2.16,
    "debug": false,
    "test": false,
    "health_record": true,
    "location_cache": true,
    "distance_unit": "km",
    "reconnecting_timeout": 15,
    "logging_color": true,
    "catch": {
      "any": {"catch_above_cp": 0, "catch_above_iv": 0, "logic": "or"},
      "// Example of always catching Rattata:": {},
      "// Rattata": { "always_catch" : true }
    },
    "release": {
      "any": {"release_below_cp": 0, "release_below_iv": 0, "logic": "or"},
      "// Example of always releasing Rattata:": {},
      "// Rattata": {"always_release": true},
      "// Example of keeping 3 stronger (based on CP) Pidgey:": {},
      "// Pidgey": {"keep_best_cp": 3},
      "// Example of keeping 2 stronger (based on IV) Zubat:": {},
      "// Zubat": {"keep_best_iv": 2},
      "// Also, it is working with any": {},
      "// any": {"keep_best_iv": 3},
      "// Example of keeping the 2 strongest (based on CP) and 3 best (based on IV) Zubat:": {},
      "// Zubat": {"keep_best_cp": 2, "keep_best_iv": 3}
    },
    "vips" : {
        "Any pokemon put here directly force to use Berry & Best Ball to capture, to secure the capture rate!": {},
        "any": {"catch_above_cp": 1200, "catch_above_iv": 0.9, "logic": "or" },
        "Lapras": {},
        "Moltres": {},
        "Zapdos": {},
        "Articuno": {},

        "// S-Tier pokemons (if pokemon can be evolved into tier, list the representative)": {},
        "Mewtwo": {},
        "Dragonite": {},
        "Snorlax": {},
        "// Mew evolves to Mewtwo": {},
        "Mew": {},
        "Arcanine": {},
        "Vaporeon": {},
        "Gyarados": {},
        "Exeggutor": {},
        "Muk": {},
        "Weezing": {},
        "Flareon": {}
    }
}

### Other Information
OS: IOS
Branch: dev
Git Commit: 86fc77478dfe85ee3609169040be825d4fa3512e
Python Version: 2.7.12

**Comment by jane0815** [comment]
Exactly the same problem for me. Also pulled a fresh clone after update did not work.

**Comment by br0kenb1nary** [comment]
It may be that your IP has been banned but the bot catches it as a timeout or improper redirect. I created a small plugin that will automatically fetch and set proxy settings for the bot based on the latitude and longitude of your player.

Check it out here and I hope it helps.
https://github.com/br0kenb1nary/PogoPlug

**Comment by yweitkemper** [comment]
Thanks ill try it out and report back.

**Comment by yweitkemper** [comment]
Says page not found for that link.

**Comment by br0kenb1nary** [comment]
Try again?

**Comment by br0kenb1nary** [comment]
Try again? https://github.com/br0kenb1nary/PogoPlugin

**Comment by jane0815** [comment]
I switched to dev and all works fine again

**Comment by yweitkemper** [comment]
already on dev, same prob :/

**Comment by NatNew0** [comment]
You have the wrong encrypt.so : "encrypt.so: mach-o, but wrong architecture',".
Try delete & create a new one or give correct location in config.

**Comment by yweitkemper** [comment]
I have been getting my encrypt from

 https://github.com/nivong/PokemonGo-Bot/blob/dev/docs/manual_installation.md 

Where would i get the correct one?

**Comment by neohunter** [comment]
I've the same problem, i dont consider necesary to post my config.json, this is the backtrace

`2016-08-18 15:19:11,297 [       cli] [INFO] Most Perfect Pokemon:
Traceback (most recent call last):
  File "pokecli.py", line 570, in <module>
    main()
  File "pokecli.py", line 91, in main
    tree = TreeConfigBuilder(bot, config.raw_tasks).build()
  File "/usr/src/app/pokemongo_bot/tree_config_builder.py", line 54, in build
    worker = self._get_worker_by_name(task_type)
  File "/usr/src/app/pokemongo_bot/tree_config_builder.py", line 21, in _get_worker_by_name
    raise ConfigException('No worker named {} defined'.format(name))
pokemongo_bot.tree_config_builder.ConfigException: No worker named RandomPause defined
2016-08-18 15:19:11,303 [sentry.errors] [ERROR] Sentry responded with an error: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128) (url: https://app.getsentry.com/api/90254/store/)
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/raven/transport/threaded.py", line 174, in send_sync
    super(ThreadedHTTPTransport, self).send(data, headers)
  File "/usr/local/lib/python2.7/site-packages/raven/transport/http.py", line 47, in send
    ca_certs=self.ca_certs,
  File "/usr/local/lib/python2.7/site-packages/raven/utils/http.py", line 66, in urlopen
    return opener.open(url, data, timeout)
  File "/usr/local/lib/python2.7/site-packages/future/backports/urllib/request.py", line 494, in open
    response = self._open(req, data)
  File "/usr/local/lib/python2.7/site-packages/future/backports/urllib/request.py", line 512, in _open
    '_open', req)
  File "/usr/local/lib/python2.7/site-packages/future/backports/urllib/request.py", line 466, in _call_chain
    result = func(*args)
  File "/usr/local/lib/python2.7/site-packages/raven/utils/http.py", line 46, in https_open
    return self.do_open(ValidHTTPSConnection, req)
  File "/usr/local/lib/python2.7/site-packages/future/backports/urllib/request.py", line 1284, in do_open
    h.request(req.get_method(), req.selector, req.data, headers)
  File "/usr/local/lib/python2.7/httplib.py", line 1057, in request
    self._send_request(method, url, body, headers)
  File "/usr/local/lib/python2.7/httplib.py", line 1097, in _send_request
    self.endheaders(body)
  File "/usr/local/lib/python2.7/httplib.py", line 1053, in endheaders
    self._send_output(message_body)
  File "/usr/local/lib/python2.7/httplib.py", line 895, in _send_output
    msg += message_body
UnicodeDecodeError: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128)
2016-08-18 15:19:11,305 [sentry.errors.uncaught] [ERROR] [u'ConfigException: No worker named RandomPause defined', u'  File "pokecli.py", line 570, in <module>', u'  File "pokecli.py", line 91, in main', u'  File "pokemongo_bot/tree_config_builder.py", line 54, in build', u'  File "pokemongo_bot/tree_config_builder.py", line 21, in _get_worker_by_name']`

**Comment by NatNew0** [comment]
@yweitkemper  Maybe you should try the setup.sh from this project.  "setup.sh -e" should build the encrypt.so . Maybe you should better start again with "setup -i" with a fresh checkout.

**Comment by MZorzy** [comment]
just git pull to dev 91ad8dfce88a3686be25eb0e2118d6715c26fad0
and same error

UnicodeDecodeError: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128)

2016-08-18 23:46:17,630 [sentry.errors.uncaught] [ERROR] [u"AttributeError: 'PokemonGoBot' object has no attribute 'get_inventory'", u'  File "pokecli.py", line 595, in <module>', u'  File "pokecli.py", line 92, in main', u'  File "pokemongo_bot/tree_config_builder.py", line 74, in build', u'  File "pokemongo_bot/base_task.py", line 20, in __init__', u'  File "pokemongo_bot/cell_workers/collect_level_up_reward.py", line 11, in initialize', u'  File "pokemongo_bot/cell_workers/collect_level_up_reward.py", line 59, in _get_current_level']

**Comment by yweitkemper** [comment]
Tried the setup.sh to same result

**Comment by NatNew0** [comment]
@yweitkemper  So you could build an encrypt.so on your system successfully? And you still get: "no suitable image found... encrypt.so: mach-o, but wrong architecture'" ? Maybe you can try manually  in the subdirectory pgoencrypt/src and run "make" . Then post the output . 
btw. what architecture are you on ? 

@MZorzy Your problem seems to be a different one. Is there a chance that you are using a completely fresh user ? Maybe you should finish the tutorial first (or try the new config option)

**Comment by yweitkemper** [comment]
I actually left the house but I'll try manually building it tonight, thanks for the tip I'll report back tonight.

**Comment by k4n30** [comment]
@yweitkemper your original issue was not having a location in your config file, your current issue is not having the correct .so file which you are already working through, let us know how you go

@neohunter while you have the same error message, it's not the same problem. You are using a task called "RandomPause" which doesn't exist in your version of the bot. This was added 19 hours ago to dev (of the the time this is posted) so I'm guessing you have old code but have pulled the task from the wiki without updating your bot properly. I'd suggest pulling the latest version and trying again. As your issue is not related to this issue, if the problem persists, please create a new issue.

@MZorzy you have a different error again (same message though) - without more logs and configs we can't help you. Can you create a new issue please

**Comment by SBS0311** [comment]
Got a similar message along with the "failed to get data" message when trying to log in through the actual app so I was figuring this is a message you get if you're banned.

**Comment by k4n30** [comment]
@SBS0311 - that's correct

**Comment by SBS0311** [comment]
Super. Not trying to be annoying but is there any way to fix this? I honestly don't have the patience or desire to start over so if this is it, so be it.

**Comment by k4n30** [comment]
Nope, risk vs reward

**Comment by buzzqw** [comment]
have you tried disabling health report ?

BHH

**Comment by yweitkemper** [comment]
So i managed to recreate the encrypt, when I use the automatic install I get the first error: 

UnicodeDecodeError: 'ascii' codec can't decode byte 0x9c in position 1: ordinal not in range(128)
2016-08-18 07:30:14,607 [sentry.errors.uncaught] [ERROR] [u'NoPlayerPositionSetException'...

When I input the config file manually i get the second error again.

PokemonGo-Bot/encrypt.so: mach-o, but wrong architecture', u'

**Comment by NatNew0** [comment]
Hi, so we closed the circle now :-) 
Maybe you should update to the latest dev version first. Then, can you verify that your config has a location set ? like     "location": "80.5749146,94.5955037,7.62z", or "location": "Paris, France" ?
What do you mean with "I input the config file manually" ? You choosed a different config file ? Normally you should run with "./run.sh" or "./run.sh -cf configs/config.json". 
Your error with the wrong architecture might be something Apple/May specific ? 
Do you see any error messages from the make command of the encrypt.so ?

**Comment by Panderine** [comment]
Try setting "health_record": false, in config.json

**Comment by thementos** [comment]
I use config.json.example it has error like this. 

```
bot1-pokego_1     | Traceback (most recent call last):
bot1-pokego_1     |   File "pokecli.py", line 578, in <module>
bot1-pokego_1     |     main()
bot1-pokego_1     |   File "pokecli.py", line 90, in main
bot1-pokego_1     |     tree = TreeConfigBuilder(bot, config.raw_tasks).build()
bot1-pokego_1     |   File "/usr/src/app/pokemongo_bot/tree_config_builder.py", line 43, in build
bot1-pokego_1     |     worker = self._get_worker_by_name(task_type)
bot1-pokego_1     |   File "/usr/src/app/pokemongo_bot/tree_config_builder.py", line 21, in _get_worker_by_name
bot1-pokego_1     |     raise ConfigException('No worker named {} defined'.format(name))
bot1-pokego_1     | pokemongo_bot.tree_config_builder.ConfigException: No worker named CatchPokemon defined
bot1-pokego_1     | pokemongobot_bot1-pokego_1 exited with code 1
```

but in tree_config_builder.py has 

```
task_type = 'CatchPokemon'
                    task_config = {}
                    self.bot.logger.warning('The CatchVisiblePokemon & CatchLuredPokemon tasks have been replaced with '
                                            'CatchPokemon.  CatchPokemon has been enabled with default settings.')
```

**Comment by thementos** [comment]
ok, I try too remove docker images and container. It work.

**Comment by yweitkemper** [comment]
Hi again , sorry was gone for the weekend. But i tried it again with the newest dev still the same error. I mean I tried using the auto setup and the manual setup with the errors. I also noticed near  the top of the second error when i use auto config it has the following not really sure what it means could that have anything to do with it?

2016-08-18 10:05:53,257 [sentry.errors.serializer] [ERROR] maximum recursion depth exceeded while calling a Python object
Traceback (most recent call last):
